docs/changes/convolution/13300.api.rst

```diff
@@ -0,0 +1,2 @@
+Removed deprecated ``MexicanHat1DKernel`` and ``MexicanHat2DKernel``
+classes.
```

Comment:
In case people been ignoring the warning, let's be clearer.
```suggestion
classes. Please use ``RickerWavelet1DKernel`` and ``RickerWavelet2DKernel`` instead.
```

Reply:
Wanna squash while you are at it? 😸

Reply:
Done and squashed.